core/client/fs/src/main/java/alluxio/client/block/stream/LocalFileDataWriter.java

```diff
@@ -135,11 +134,11 @@ public void writeChunk(final ByteBuf buf) throws IOException {
 
   @Override
   public void cancel() throws IOException {
-    if (mStream.isClosed() || mStream.isCanceled()) {
-      return;
-    }
+    boolean cancelStream = !(mStream.isClosed() || mStream.isCanceled());
```

Comment:
Perhaps extract this logic to another method?

Reply:
i removed the check all together as the stream already has an `isOpen` check for both `close` and `cancel`. updated the javadoc to reflect that